Tests.Integration.Presentation.Web/Contract/V2/ItContractsApiV2Test.cs

```diff
@@ -745,10 +745,11 @@ public async Task Can_PATCH_With_GeneralData()
 
             //Act - reset
             var resetRequest = new ContractGeneralDataWriteRequestDTO();
+            //Is correct?
+            resetRequest.Validity = new ValidityWriteRequestDTO();
```

Comment:
let's talk about this test. It needs to be extended a bit more to prove the new resolution

Reply:
@a-naskret This question actually made me investigate why it was necessary. And as it turns out there was an error in the base mapper which did not allow clients to do a root level reset with General: null

It does now :-)